vuepress/docs/next/tutorials/solution/strapi.md

```diff
@@ -0,0 +1,194 @@
+
+### Create Content in Strapi
+
+Before you can configure the Strapi Content Template Widget, you must add content to your Strapi instance. Refer to the Strapi [Quick Start Guide](https://docs.strapi.io/developer-docs/latest/getting-started/quick-start.html) for a tutorial on content creation. Additional information is available in the [Strapi user guide](https://docs.strapi.io/user-docs/latest/getting-started/introduction.html).
+
+### Configure the Strapi Config Widget
+
+The Strapi Config Widget provides the Strapi Content Template, Strapi Content and Strapi Content List widgets with the application URL of the Strapi instance. The URL is managed from a single field entry, which must be defined prior to manipulating content. 
+
+Follow the steps below to publish the Strapi Config Widget to a page in your Entando Application and expose the Strapi URL.
+
+1. [Create a page](../compose/page-management.md#create-a-page) in your Entando Application dedicated to the Strapi Config Widget
+2. Go to  `App Builder` → `Pages` → `Management`
+3. Find the Strapi configuration page in the page tree and click on the three dots representing the `Actions` icon
+4. Select `Design` from the drop-down
+5. Click on the `Widgets` tab in the right panel and expand the `User` section
+6. Drag and drop the Strapi Config Widget into an empty frame in the middle panel
+
+![strapi-config-page-designer.png](./strapi-images/strapi-config-page-designer.png)
+
+7. Click `Publish`
+8. Click `View Published Page`
+9. On the published page, enter the URL of the Strapi instance, which is your App Builder base URL followed by `/entando-strapi`, e.g. `http://hubdev.okd-entando.org/entando-strapi`
+
+![strapi-url.png](./strapi-images/strapi-url.png)
+
+### Configure the Strapi Content Template Widget
+
+To manage Strapi content in your Entando Application, the content must belong to a type that is assigned attributes via a template. Follow the steps below to create and modify content templates with the Strapi Content Template Widget.
+
+#### Create a Content Template
+
+1. [Create a page](../compose/page-management.md#create-a-page) in your Entando Application dedicated to the Strapi Content Template Widget
+2. Go to  `App Builder` → `Pages` → `Management`
+3. Find the Strapi template page in the page tree and click on the `Actions` icon 
+4. Select `Design` from the drop-down
+5. Click on the `Widgets` tab in the right panel and expand the `User` section
+6. Drag and drop the Strapi Content Template Widget into an empty frame in the middle panel
+
+![strapi-template-page-designer.png](./strapi-images/strapi-template-page-designer.png)
+
+7. Click `Publish`
+8. Click `View Published Page`
+9. Click `Add` on the right
+
+![strapi-add-template.png](./strapi-images/strapi-add-template.png)
+
+10. Add a content template, where the following fields are mandatory:
+   - Content type: The drop-down displays [existing content types](#create-content-in-strapi)
```

Comment:
is there any guidance on what do put in for these fields?

Reply:
the user created this content earlier; it is at their discretion. they should have a clear understanding of what's logical and permissible